stripe_block.py

```diff
@@ -50,7 +49,7 @@ class WebServer(PropertyHolder):
 
     host = StringProperty(title='Host', default='0.0.0.0')
     port = IntProperty(title='Port', default=8182)
-    endpoint = StringProperty(title='Endpoint', default='')
+    endpoint = StringProperty(title='Endpoint', default="", allow_none=True)
```

Comment:
I don't believe that this is valid: the endpoint is required by `RESTHandler`, which is inherited by the `BuildSignal` class. Why do we need to allow `None`?